lms/views/api/jstor.py

```diff
@@ -8,16 +8,18 @@
 class JSTORAPIViews:
     def __init__(self, request):
         self.request = request
-        self.jstor_service = request.find_service(iface=JSTORService)
+        self.jstor_service: JSTORService = request.find_service(iface=JSTORService)
 
     @view_config(route_name="jstor_api.articles.metadata")
     def article_metadata(self):
         article_id = self.request.matchdict["article_id"]
         article_info = self.jstor_service.metadata(article_id)
+        content_status = article_info["content_status"]
 
-        is_collection = False  # Placeholder
-
-        return {"title": article_info["title"], "is_collection": is_collection}
+        return {
+            "title": article_info["title"],
+            "content_status": content_status.name.lower(),
+        }
```

Comment:
In the VitalSource work we've tried to keep the proxy API as thin a layer over the service as possible, which means pushing down the format we'd like all the way back to the service / client.

There are reasons not to do this, for example if you have a single call used in many different ways, but quite a few of our methods only exist to serve these proxy end-points directly.

With any of the suggestions that result in something directly serialisable from above, we should be able to change this method to be:

```python
    @view_config(route_name="jstor_api.articles.metadata")
    def article_metadata(self):
        return self.jstor_service.metadata(self.request.matchdict["article_id"])
```

Which is nice and snappy and also makes testing a breeze.